lib/std/math/log.zig

```diff
@@ -23,14 +23,17 @@ pub fn log(comptime T: type, base: T, x: T) T {
         .ComptimeFloat => {
             return @as(comptime_float, @log(@as(f64, x)) / @log(float_base));
         },
+
```

Comment:
Please add, that we should special case comptime-known divisions of power of two with shifts and/or make an issue.